arcgis-ios-sdk-samples/Edit data/Edit feature attachments/AttachmentsTableViewController.swift

```diff
@@ -0,0 +1,174 @@
+// Copyright 2016 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class AttachmentsTableViewController: UITableViewController {
+    
+    weak var feature: AGSArcGISFeature?
+    private var attachments: [AGSAttachment] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        loadAttachments()
```

Comment:
There really isn't a good reason to tie the loading of attachments to the loading of the view controller's view. I would suggest calling this in a `didSet` observer of `feature`.